src/postgres/commands/createPostgresServer/steps/PostgresServerNameStep.ts

```diff
@@ -46,11 +47,10 @@ async function validatePostgresServerName(name: string, client: PostgreSQLManage
         return localize('serverNamePrefixSuffixCheck', 'Server name must not start or end in a hyphen.');
     }
 
-    const availabilityRequest: NameAvailabilityRequest = { name: name, type: "Microsoft.DBforPostgreSQL" };
-    const availability: NameAvailability = (await client.checkNameAvailability.execute(availabilityRequest));
+    const availability: AbstractNameAvailability = await client.checkNameAvailability.execute({name: name, type: "Microsoft.DBforPostgreSQL"});
 
     if (!availability.nameAvailable) {
-        if (availability.reason === 'AlreadyExists') {
+        if (availability.message === 'AlreadyExists') {
```

Comment:
I don't think `message` is right - this is what I'm seeing
<img width="487" alt="Screen Shot 2021-06-30 at 1 12 03 PM" src="https://user-images.githubusercontent.com/11282622/124025017-ce10bf00-d9a4-11eb-9100-daa35abf7a37.png">

That's for single server. Not sure what flexible looks like because of the sdk issue.